### PR TITLE
Added Permissions for private messages - based on UID

### PR DESCRIPTION
**Pull request by Najsr**
I hope this will be enough for start.

### CODE REVIEW

**Comment by Najsr** [comment]
Simplified the code, check it out now ;)

File: app/TeamSpeak3Bot.php [hunk 1]
```diff
@@ -425,6 +425,9 @@ public function onMessage(Event $event)
                 }
 
                 if ($event["msg"]->startsWith($trigger)) {
+                    if ($data['targetmode'] == "1" && !in_array($data['invokeruid'], @$config->CONFIG['uids'])) {
```
Comment (JABirchall) [inline app/TeamSpeak3Bot.php]: Permisison would be prefered to cover all text channels for individual plugins

File: config/plugins/MD5.json [hunk 2]
```diff
@@ -6,5 +6,9 @@
   "triggers": [
     "!md5"
   ],
+  "uids": [
```
Comment (JABirchall) [inline config/plugins/MD5.json]: Permission are required to be based on the invoker server groups.

**Comment by JABirchall** [comment]
Please submit a new pull request with the requested changes.

**Pull request closed without merging**